### ISSUE TITLE
MultinomialNB is not supported

### ISSUE THREAD

**Issue by casertap** [TeamHG-Memex/eli5#131]
Hi,
great work with eli5, this is awesome, thanks.

MultinomialNB is not supported but I am wondering why.
Is this classifier so different than other classifiers that it is impossible to have it in eli5?

Thanks,

**Comment by kmike** [comment]
Hey @casertap,

It is definitely possible to support it (and we should do that), someone just have to figure out the details - check https://github.com/TeamHG-Memex/eli5/issues/81. Do you want to give it a try? I think it is a matter of finding a way to make MultinomialNB.coef_ compatible with other linear classifiers.

I'm closing it in favor of #81.
